packages/transmute-adapter-firestore/__tests__/interface.spec.js

```diff
@@ -1,13 +1,14 @@
 const TransmuteAdapterFirestore = require("../index");
 
-const admin = require("firebase-admin");
+// const admin = require("firebase-admin");
+// const serviceAccount = require("../service-account-firebase-adminsdk.json")
+// admin.initializeApp({
+//   credential: admin.credential.cert(serviceAccount)
+// });
+// const db = admin.firestore();
 
-const serviceAccount = require("../serviceAccount");
+let db = {};
 
```

Comment:
is it possible just to pass an empty object into `TransmuteAdapterFirestore({}, "events")` and remove `let db = {}` since its not reference within the file again.

Reply:
yep, it just left over from the mocking process